scripts/verify-registry-install.sh

```diff
@@ -2,7 +2,7 @@
 
 # NOTE: MUST BE RAN *AFTER* THE PUBLISH SUITE
 
-export REGISTRY="${ARTIFACTORY_URL}/npm-topic"
+export PUBLISHED_REGISTRY="${ARTIFACTORY_URL}/npm-topic"
```

Comment:
Should this be `PUBLISH_REGISTRY` to match the one in `publish.sh` with the same value?